src/main/scala/org/hatdex/dex/apiV3/services/DexClient.scala

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (C) 2016 HAT Data Exchange Ltd - All Rights Reserved
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * Written by Andrius Aucinas <andrius.aucinas@hatdex.org>, 2 / 2017
+ *
+ */
+
+package org.hatdex.dex.apiV3.services
+
+import javax.inject.Inject
+
+import org.hatdex.dex.api.services.{ DexDataPlugs, DexNotices }
+import play.api.libs.ws.WSClient
+
+class DexClient(
+    val ws: WSClient,
+    val dexAddress: String,
+    override val schema: String) extends DexOffers
+  with DexNotices
+  with DexDataPlugs
+  with DexStats
+  with DexUsers
+  with DexApplications {
+
+  override val apiVersion: String = "v3"
+  @Inject def this(ws: WSClient, dexAddress: String) = this(ws, dexAddress, "https://")
```

Comment:
I see the `http`/`https` scheme defined in the `.conf` files for many of the other components.

No biggie, just consistency.

Reply:
good suggestion however its a lib so no .conf file
It takes the protocol as a parameter this here is a secondary constructor
I guess the idea is it will be set in the conf of the app and then passed however i did not create this library so im just guessing

Reply:
Given the current drive toward encryption, it best option would be to always default to the https protocol. But it requires the whole local development environment to be able to run on https.